### DISCUSSION TITLE
Any way to import Spotify tracks?

### DISCUSSION THREAD

**Discussion by xyloflake** [navidrome/navidrome#3094] [Q&A] [answered]
https://www.navidrome.org/docs/usage/external-integrations/
I found out this and configured it for Spotify. It looks like it's only for images and something else but no tracks. So how do I import tracks?

**Comment by dweymouth** [comment, marked as answer]
You cannot import tracks from Spotify. Navidrome is for supplying your own music files

**Reply by xyloflake** [reply]
So where do I get those music files? Do I have to download them?

**Reply by dweymouth** [reply]
That's on you to figure out where you will get your music from. A lot of Navidrome users like Bandcamp to buy albums from their favorite artists in FLAC format.

**Reply by xyloflake** [reply]
> That's on you to figure out where you will get your music from. A lot of Navidrome users like Bandcamp to buy albums from their favorite artists in FLAC format.

Gotchu, thanks!
